_data/simple-icons.json

```diff
@@ -1695,6 +1695,11 @@
             "source": "https://aws.amazon.com/architecture/icons",
             "guidelines": "https://aws.amazon.com/architecture/icons"
         },
+        {
+            "title": "Awwwards",
+            "hex": "222222",
```

Comment:
But why `#222222` and not `#000000` from the logo itself?

Reply:
It is the primary color as defined in their website CSS stylesheet

Reply:
Okay then! Good to go 🎉